core/tests/webdriverio_utils/ExplorationPlayerPage.js

```diff
@@ -54,7 +54,8 @@ var ExplorationPlayerPage = function() {
   var lessonInfoTooltip = $('.e2e-test-close-lesson-info-tooltip');
   var gotItButton = $('.e2e-test-learner-got-it-button');
   var infoCardRating = $('.e2e-test-info-card-rating');
-  var nextCardButton = $('.e2e-test-continue-to-next-card-button');
+  var continueToNextCardButton = $('.e2e-test-continue-to-next-card-button');
+  var nextButton = $('.e2e-test-next-button');
```

Comment:
The `continueToNextCardButton` & `nextButton` togther creates confusion, can we call the 2nd one bit different, maybe use "forward" word as we are not going to a new/next card instead forwarding through the history? (not sure!)

Reply:
Yeah, makes total sense. I thought about it as well, just couldn't come up with a proper name. Forward sounds reasonable.